src/cursor.cr

```diff
@@ -284,11 +284,11 @@ module Clang
     # end
 
     def raw_comment_text
-      Clang.string(LibC.clang_Cursor_getRawCommentText(self))
+      LibC.clang_Cursor_getRawCommentText(self).try{|c| c.data.null? ? nil : Clang.string(c) }
```

Comment:
The following would be more idiomatic and fit better with the rest of project:
```crystal
if ptr = LibC.clang_Cursor_getRawCommentText(self)
  Clang.string(ptr)
end
```

Reply:
You're right and I find this version way more readable :+1:  I pushed the change.

Reply:
I don't see the new commit ?

Reply:
Sorry for this, looks like I pushed on the wrong remote 😳
It should be fixed by now 😄